identifiers/country-mx.csv

```diff
@@ -0,0 +1,334 @@
+id,name
+ocd-division/country:mx,México

```

Comment:
For consistency please use 'Mexico' instead.

Reply:
Source data was inconsistent with accents.